libs/libs-pomfix-run/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "pom-fix -d ../../"
+    "test": "node target/pom-fix --dir ../../"
```

Comment:
Before we were using the fact that yarn would generate a sh/bat script to execute. Now we have to invoke it manually with node because we are using the js directly.